### DISCUSSION TITLE
Node crash on computer sleep - any way to avoid this?

### DISCUSSION THREAD

**Discussion by astrohm** [phcode-dev/phoenix#1790] [Q&A] [answered]
From what I can tell, every time my computer (Dell running Windows 10) goes into sleep mode, when I wake it and return to Phoenix Code (desktop), I get an "Oops! Something went wrong!" message with the error code ERROR_NODE_JS_CRASH. This wasn't the case when I first installed the Phoenix desktop app, it started in May or June, and I can't think of anything that changed in my environment. Has anyone else seen this, or does anyone have any ideas to solve this issue?

**Comment by abose** [comment, marked as answer]
Thanks for reporting the issue.  Created a high priority issue to track to address this: https://github.com/phcode-dev/phoenix/issues/1791

This will be addressed in the next release.

NB: We have not got any abnormal devieations in crash reports for this specific error between May and June. More details can be found in the associated issue.

**Reply by abose** [reply]
Issue is fixed and will be available in the September release planned this week.

**Comment by mbsankar84** [comment]
![image](https://github.com/user-attachments/assets/b3d3dadd-54e2-44c2-8302-d8df7edb8238)
I just downloaded in october & worked on a project. When system sleep n re open it asked to recover files. I accepted recover. it's shown like this entire page as corrupted DOTS. Tried to open the same with brackets thrown " Unknown encode while open file" error. What to do? No code there either new or old changes. Lost all code? Can we get a very quick reply on this how to resolve?

**Reply by abose** [reply]
Apologies for the issue. This is tracked in https://github.com/phcode-dev/phoenix/issues/1885 and we need more information to investigate how/why this happened. Please comment on the above issue to help us fix the issue.
